src/plugins/maps_legacy/public/map/_leaflet_overrides.scss

```diff
@@ -1,164 +1,166 @@
+.visualization {
+  .leaflet-touch .leaflet-bar,
+  .leaflet-draw-actions {
+    @include euiBottomShadowMedium($color: $euiShadowColorLarge, $opacity: 0.2);
 
-  border: none;
-}
-
-.leaflet-container {
-  background: $euiColorEmptyShade;
-
-  // the heatmap layer plugin logs an error to the console when the map is in a 0-sized container
-  min-width: 1px !important;
-  min-height: 1px !important;
-}
-
-.leaflet-clickable {
-  &:hover {
-    stroke-width: $euiSizeS;
-    stroke-opacity: 0.8;
+    border: none;
   }
-}
 
-/**
- * 1. Since Leaflet is an external library, we also have to provide EUI variables
- *    to non-override colors for darkmode.
- */
+  .leaflet-container {
+    background: $euiColorEmptyShade;
 
-.leaflet-draw-actions,
-.leaflet-control {
-  a {
-    background-color: lightOrDarkTheme($euiColorEmptyShade, $euiColorLightShade); /* 1 */
-    border-color: lightOrDarkTheme($euiColorLightShade, $euiColorMediumShade) !important; /* 1 */
-    color: $euiTextColor !important; /* 1 */
+    // the heatmap layer plugin logs an error to the console when the map is in a 0-sized container
+    min-width: 1px !important;
+    min-height: 1px !important;
+  }
 
+  .leaflet-clickable {
     &:hover {
-      background-color: $euiColorLightestShade;
+      stroke-width: $euiSizeS;
+      stroke-opacity: 0.8;
     }
   }
-}
 
-.leaflet-touch .leaflet-bar a:first-child {
-  border-top-left-radius: $euiBorderRadius;
-  border-top-right-radius: $euiBorderRadius;
-}
+  /**
+  * 1. Since Leaflet is an external library, we also have to provide EUI variables
+  *    to non-override colors for darkmode.
+  */
+
+  .leaflet-draw-actions,
+  .leaflet-control {
+    a {
+      background-color: lightOrDarkTheme($euiColorEmptyShade, $euiColorLightShade); /* 1 */
+      border-color: lightOrDarkTheme($euiColorLightShade, $euiColorMediumShade) !important; /* 1 */
+      color: $euiTextColor !important; /* 1 */
+
+      &:hover {
+        background-color: $euiColorLightestShade;
+      }
+    }
+  }
 
-.leaflet-touch .leaflet-bar a:last-child {
-  border-bottom-left-radius: $euiBorderRadius;
-  border-bottom-right-radius: $euiBorderRadius;
-}
+  .leaflet-touch .leaflet-bar a:first-child {
+    border-top-left-radius: $euiBorderRadius;
+    border-top-right-radius: $euiBorderRadius;
+  }
 
-.leaflet-retina .leaflet-draw-toolbar a {
-  background-image: url($visMapLeafletSprite); /* 1 */
-}
+  .leaflet-touch .leaflet-bar a:last-child {
+    border-bottom-left-radius: $euiBorderRadius;
+    border-bottom-right-radius: $euiBorderRadius;
+  }
 
-.leaflet-control-layers-expanded {
-  padding: 0;
-  margin: 0;
+  .leaflet-retina .leaflet-draw-toolbar a {
+    background-image: url($visMapLeafletSprite); /* 1 */
+  }
 
-  @include fontSize(11px);
+  .leaflet-control-layers-expanded {
+    padding: 0;
+    margin: 0;
 
-  font-family: $euiFontFamily;
-  font-weight: $euiFontWeightMedium;
-  line-height: $euiLineHeight;
+    @include fontSize(11px);
```

Comment:
Would love to use some variant of `euiSize` here, but I think that's outside the scope of this PR

Reply:
Yeah, agreed. I imagine we'll tackle such things as part of https://github.com/opensearch-project/oui/issues/817